src/components/_classes/component/Component.js

```diff
@@ -1836,16 +1836,18 @@ export default class Component extends Element {
   }
 
   addAce(element, settings, onChange) {
-    settings = _.merge(_.get(this.options, 'editors.ace.settings', {}), settings || {});
```

Comment:
`_.merge` mutates object that is passed as first argument, so `options` passed to `Formio` are getting mutated, which is bad